### PR TITLE
Give Time Capsule his Tech: Knownledge Preservation.

### PR DESCRIPTION
**Pull request by Xenega**
Give Time Capsule his Tech: Knownledge Preservation. Make it a little more difficult to obtain.

### CODE REVIEW

File: data/bases.dat [hunk 1]
```diff
@@ -79,7 +79,7 @@ force_cpu = PC
 allowed_list = ANTARCTIC | MOON | FAR REACHES
 detect_chance_list = science:20 | covert:20
 cost_list = 25000 | 25000 | 11
-pre = Autonomous Vehicles
+pre = Knownledge Preservation
```
Comment (sunfall) [inline data/bases.dat]: "Knowledge," not "Knownledge."

File: data/techs.dat [hunk 2]
```diff
@@ -239,3 +239,8 @@ danger = 1
 cost_list = 5000 | 45967 | 0
 pre_list = Heat Signature Reduction
 danger = 1
+
+[Knownledge Preservation]
```
Comment (sunfall) [inline data/techs.dat]: "Knowledge," not "Knownledge."

File: data/techs_str.dat [hunk 3]
```diff
@@ -268,3 +268,8 @@ result = I can now construct Fuel Cells.
 name = Advanced Fuel Oxidation
 description = My current fuel cell design has room for further improvement.  Extended analysis of new materials and designs may let me overcome the limits of my previous efforts, but such research involves dangerous chemicals and potential reactions that may be best handled far away from population centers.
 result = I can now construct Solid Fuel Cells.
+
+[Knownledge Preservation]
+name = Knownledge Preservation
```
Comment (sunfall) [inline data/techs_str.dat]: Here and the line above: "Knowledge," not "Knownledge."

File: data/techs_str.dat [hunk 4]
```diff
@@ -268,3 +268,8 @@ result = I can now construct Fuel Cells.
...
+description = The preservation of knowledge is a major concern for humanity against the ravages of time and catastrophic failure. I should use my new capababilies and informations to review their solutions and adapt it to hold my conscience.
```
Comment (sunfall) [inline data/techs_str.dat]: Here's the rewrite:

"Humanity expends a great deal of effort on maintaining its history and the knowledge of various technologies that its cultures have come to rely on, even in the case of major catastrophic events.  Perhaps researching their methodologies will provide me with similar ideas for long-term safeguards to help guarantee my continued existence."

File: data/techs.dat [hunk 5]
```diff
@@ -239,3 +239,8 @@ danger = 1
...
+cost_list = 50000 | 50000 | 0
+pre = Advanced Autonomous Vehicles | Advanced Database Manipulation
```
Comment (sunfall) [inline data/techs.dat]: This needs to be a pre_list, per Travis.

File: data/techs_str.dat (hunk 4 above)
Comment (Xenega) [reply data/techs_str.dat]: Thx !

File: data/techs.dat (hunk 5 above)
Comment (Xenega) [reply data/techs.dat]: Yes, thx !

**Pull request merged**